pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -153,7 +153,7 @@ func (s *Server) RunCA(grpc *grpc.Server, ca caserver.CertificateAuthority, opts
 	if err == nil {
 		tok, err := detectAuthEnv(string(token))
 		if err != nil {
-			log.Warn("Starting with invalid K8S JWT token", err, string(token))
+			log.Warnf("Starting with invalid K8S JWT token: %v", err)
```

Comment:
I really don't like more Warnf and equivalents. 

Much better to add a structured logging method to the log package - for example WarnCtx() to match the new API - and convert the existing Warn() to WarnCtx.